styles.css

```diff
@@ -408,6 +428,115 @@ textarea {
   color: var(--clr-white);
 }
 
+/* PopUp Window */
+.feature {
+  background-color: #fff;
+  width: 60%;
+  height: 95%;
+  padding: 0 2rem;
+  overflow-y: scroll;
+  border-radius: 16px;
+}
```

Comment:
- [ ] Your popup modal is great and you did a great job implementing the suggested changes. But the text in the buttons on the mobile version is not aligned after your changes. I suggest you play around with the padding, width, and font-size of your modal and elements inside so they can look more like the template.

 [OPTIONAL] Your modal cannot display the entire content on the screen and the user have to scroll down to look for the buttons. This is not the best user experience and would be nicer if everything in the modal is possible to see right after the modal is opened. Try playing around with the dimension of your modal(width), reducing the padding on the sides, changing the font-sizes of the elements inside, and making sure the values match with the values from the Figma template.
 -Your mobile modal:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/61919617/189106561-95e7630c-6ff4-47bb-92f3-f87645b2c920.png">